train/sft.sh

```diff
@@ -1,37 +1,41 @@
+# Reference Running: bash train/sft.sh
+# {'train_runtime': 5268.8407, 'train_samples_per_second': 0.949, 'train_steps_per_second': 0.119, 'train_loss': 0.1172730620391667, 'epoch': 5.0}
 uid="$(date +%Y%m%d_%H%M%S)"
 base_model="Qwen/Qwen2.5-32B-Instruct"
 lr=1e-5
 min_lr=0
 epochs=5
-micro_batch_size=2 # -> batch_size will be 16 if 8 gpus
-push_to_hub=true
-gradient_accumulation_steps=2
+weight_decay=1e-4 # -> the same training pipe as slurm_training
+micro_batch_size=1 # -> batch_size will be 8 if 8 gpus
```

Comment:
```suggestion
micro_batch_size=1 # -> batch_size will be 16 if 16 gpus
```

(I thin we did 2 nodes; 16 gpus)